EIPS/eip-8888.md

```diff
@@ -0,0 +1,194 @@
+| `chainId`                 | unique identifier for the blockchain.                                            |
+| `<hardfork(Block\|Time)>` | block height or timestamp to activate the named hardfork.                        |
+| `terminalTotalDifficulty` | difficulty after which to switch from PoW to PoS.                                |
+| `depositContractAddress`  | Ethereum address for the deposit contract                                        |
+| `blobSchedule`            | Map of hardforks and their EIP-4844 DAS [configuration parameters](eip-7840.md). |
+
+### `blobSchedule` Object
+
+| Field                   | Description                                  |
+|-------------------------|----------------------------------------------|
+| `target`                | desired number of blobs to include per block |
+| `max`                   | maximum number of blobs to include per block |
+| `baseFeeUpdateFraction` | input to pricing formula per EIP-4844        |
+
+### `alloc` Object
+
+The `alloc` field defines the initial state at genesis. It maps addresses (as lowercase hex strings) to the following object:
+
+| Field          | Description                                     |
+|----------------|-------------------------------------------------|
+| `balance`      | decimal balance in wei.                         |
+| `code`         | Hex-encoded EVM bytecode.                       |
+| `nonce`        | decimal value.                                  |
+| `storage`      | Key-value hex map representing initial storage. |
+
+
+
+JSON Schema
+
+
+```json
+
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$defs": {
+  	"hexOrDecimal48": {
+    		"type": "string",
+    		"pattern": "^(0x[0-9a-fA-F]{1,12}|[0-9]{1,12})$"
+  	},
+    "address": {
+      "type": "string",
+      "pattern":  "^0x[0-9a-fA-F]{40}$"
+    },
+    "hash": {
+      "type": "string",
+      "pattern": "^0x[0-9a-fA-F]{64}$"
+    }
+  },
+  "title": "Ethereum Genesis File",
+  "type": "object",
+  "required": ["alloc", "gasLimit", "difficulty"],
+  "properties": {
+    "config": {
+      "type": "object",
+      "properties": {
+        "chainId": { "type": "integer" },
+        "homesteadBlock": { "type": "integer" },
+        "daoForkBlock": { "type": "integer" },
+        "eip150Block": { "type": "integer" },
+        "eip155Block": { "type": "integer" },
+        "eip158Block": { "type": "integer" },
+        "byzantiumBlock": { "type": "integer" },
+        "constantinopleBlock": { "type": "integer" },
+        "petersburgBlock": { "type": "integer" },
+        "istanbulBlock": { "type": "integer" },
+        "muirGlacierBlock": {"type": "integer"},
+        "berlinBlock": { "type": "integer" },
+        "londonBlock": { "type": "integer" },
+        "arrowGlacierBlock": { "type": "integer" },
+        "grayGlacierBlock": { "type": "integer" },
+        "terminalTotalDifficulty": { "type": "integer" },
+        "mergeNetsplitBlock": { "type": "integer"},
```

Comment:
This needs documentation because this is confusing. I remember I researched this a month or so back but I now don't completely recall what this was. I think this had to do with a forkID to signal if you are merge-ready or not.

Reply:
For mainnet this block seems to be unset: https://github.com/ethereum/go-ethereum/blob/63740b7aca732f6c204c56a9e4b4932f5251e4e0/params/config.go#L60 

So maybe we can remove this